common-modules/src/main/java/com/softserve/if072/common/model/Cart.java

```diff
@@ -66,4 +66,80 @@ public String toString() {
                 product.getUnit().getName() +
                 "\n}";
     }
+
+    public static class Builder {
```

Comment:
Better to move this to the test classes. I mean to put under src/test/java/com/softserve/if072/common/model/CartBuilder.java